fem/lininteg.cpp

```diff
@@ -675,14 +682,14 @@ void DGElasticityDirichletLFIntegrator::AssembleRHSElementVect(
       }
       else
       {
-         CalcOrtho(Tr.Face->Jacobian(), nor);
+         CalcOrtho(Tr.Jacobian(), nor);
       }
 
       double wL, wM, jcoef;
       {
          const double w = ip.weight / Tr.Elem1->Weight();
-         wL = w * lambda->Eval(*Tr.Elem1, eip);
-         wM = w * mu->Eval(*Tr.Elem1, eip);
+         wL = w * lambda->Eval(Tr, ip);
+         wM = w * mu->Eval(Tr, ip);
```

Comment:
This is not correct. Both `lambda` and `mu` are "volume" coefficients, so you want to evaluate them as such. One reason for that is the attribute -- as "volume" coefficients they expect element attributes, not boundary attributes.

Some coefficients can work as "volume", "boundary", or other types but not all coefficients support that.

Reply:
We can revert these but we need to document this to avoid confusion.